cmd/soroban-rpc/internal/db/transaction.go

```diff
@@ -135,43 +139,23 @@ func (txn *transactionHandler) trimTransactions(latestLedgerSeq uint32, retentio
 	return err
 }
 
-// GetLedgerRange pulls the min/max ledger sequence numbers from the database.
+// GetLedgerRange pulls the min/max ledger sequence numbers from the transactions table.
 func (txn *transactionHandler) GetLedgerRange(ctx context.Context) (ledgerbucketwindow.LedgerRange, error) {
```

Comment:
@aditya1702 if we are refactoring `GetLedgerRange` to fetch entries from LCM table, we def want to move this to LedgerReader.  and then we want to pass `LedgerReader` in related RPCs instead of `TxnReader`

Reply:
@psheth9 I am still refactoring things 😅 . Doing it in small small commits